### PR TITLE
Added Multilayer Perceptron (sklearn)

### PR DESCRIPTION
**Pull request by QuantumNovice**
black + flake8

### CODE REVIEW

**Comment by cclauss** [comment]
https://github.com/TheAlgorithms/Python/blob/master/Travis_CI_tests_are_failing.md

**Comment by cclauss** [comment]
Please add a test that appears in our automated test results...
https://travis-ci.com/TheAlgorithms/Python/builds/139021093#L426-L429

File: machine_learning/multilayer_perceptron_classifier.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    solver="lbfgs", alpha=1e-5, hidden_layer_sizes=(5, 2), random_state=1
+)
+
+clf.fit(X, y)
+
+
+test = [[0.0, 0.0], [0.0, 1.0], [1.0, 1.0]]
+Y = clf.predict(test)
+
+assert list(Y), [0, 0, 1]
```
Comment (cclauss) [inline machine_learning/multilayer_perceptron_classifier.py]: __assert list(Y) == [0, 0, 1]__

Comment (QuantumNovice) [reply machine_learning/multilayer_perceptron_classifier.py]: on it.

Comment (QuantumNovice) [reply machine_learning/multilayer_perceptron_classifier.py]: Done

**Comment by QuantumNovice** [comment]
> 
> 
> Please add a test that appears in our automated test results...
> https://travis-ci.com/TheAlgorithms/Python/builds/139021093#L426-L429

Done.
https://travis-ci.com/TheAlgorithms/Python/builds/139023262#L430

**Pull request merged**